### ISSUE TITLE
[TagPicker][Accessibility]Tag Picker does not read out the names of selected tag in edge

### ISSUE THREAD

**Issue by aadhars** [OfficeDev/office-ui-fabric-react#3625]
### Bug Report

- __Package version(s)__: Latest
- __Browser and OS versions__: **Edge** It works on IE

#### Priorities and help requested (not applicable if asking question):

Requested priority: (Blocking, High, Normal, Low) : High

Products/sites affected: (if applicable) VSTS

#### Describe the issue:

Tag Picker does not read out the names of selected tag in edge. It does read out in IE but does not read out in edge

#### Expected behavior:

Tag Picker should read out the selected tag in edge as well.

### If applicable, please provide a codepen repro:

It can be observed on office fabric site as well.

https://developer.microsoft.com/en-us/fabric#/components/pickers

**Comment by joschect** [comment]
This was fixed when an aria label was added to the tag picker item
